docs/roadmap.rst

```diff
@@ -0,0 +1,111 @@
+##########################################
+Rubin Science Platform Status and Roadmap
+##########################################
+
+What can you do with the Rubin Science Platform?
+================================================
+
+Below you will find a detailed (but not exhaustive) list of current and future RSP capabilities.
+
+Please note the difference between Data Previews (DP0, DP1, DP2), which are early access programs before the start of the survey, and Data Releases (DR1, DR2, etc.), which are the official data releases after the survey starts.
```

Comment:
Tighten up? There are two "which" clauses.

Reply:
There are also two objects (Data Previews and Data Releases) and hence two relative clauses (one for each) so I think this is good. It's like "I have two friends - Marie, who is French, and Xuqin, who is Chinese."